src/app/pages/model-3d/model-3d-create/model-3d-create.page.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, OnInit, inject, effect } from '@angular/core';
+import { GeneralContainerComponent } from '@components/layout';
+import { CommonModule } from '@angular/common';
+import { MatIcon } from '@angular/material/icon';
+import { Router } from '@angular/router'; 
+import {
+  FormControl,
+  ReactiveFormsModule,
+  Validators,
+  FormBuilder,
+  FormGroup,
+} from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatSelectModule } from '@angular/material/select';
+import { MatButtonModule } from '@angular/material/button';
+import { AlertService, FormsService, I18nService } from '@services/general';
+import { PagesUrlsEnum } from '@common/enums';
+import { AlertTypeEnum } from '@common/enums';
+import {ImageUploadHttpService, Model3DCreateHttpService} from '@services/http';
+
+
+@Component({
+  selector: 'app-model-3d-create',
+  templateUrl: './model-3d-create.page.html',
+  styleUrls: ['./model-3d-create.page.scss'],
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatSelectModule,
+    MatButtonModule,
+    GeneralContainerComponent,
+    MatIcon 
+  ],
+})
+export class Model3DCreatePage implements OnInit {
+  form: FormGroup;
+  animals: { id: number; name: string }[] = [];
```

Comment:
Podria ser mejor esto una interfaz 


```suggestion
  animals: IMiInterfazQueDefineElTipoYMeDiceExactamenteQueEs}[] = [];
```